src/main/java/seedu/mtracker/instrument/InstrumentManager.java

```diff
@@ -22,6 +22,14 @@ public static InstrumentManager getInstance() {
         return instruments;
     }
 
+    public int getSize() {
+        return instruments.size();
+    }
+
+    public String getInstrumentWithFileFormat(int instrumentIndex) {
+        return instruments.get(instrumentIndex).textFileFormatting();
+    }
+

```

Comment:
Remove this method and create a new class in fileManager that does all the conversion instead